Gemfile

```diff
@@ -3,4 +3,12 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
+gem 'dotenv', groups: %i[development test]
```

Comment:
My personal opinion is that the dotenv gem causes more confusion than help in a docker environment. I have not seen it behaving as expected, I think the env variables get overlayed in the ordered their files are entered in the docker-compose.yml for that project.